index.html

```diff
@@ -487,14 +510,15 @@ <h3 id="installation-sec">
           Privacy and security considerations
         </h3>
         <p>
-          During the <a>installation process</a>, it is RECOMMENDED that the
-          user agent allow the end-user to inspect the icon, name, <a>start
-          URL</a>, origin, etc. pertaining to a web application. This is to
-          give an end-user an opportunity to make a conscious decision to
-          approve, and possibly modify, the information pertaining to the web
-          application before installing it. This also gives the end-user an
-          opportunity to discern if the web application is spoofing another web
-          application, by, for example, using an unexpected icon or name.
+          During the <a data-lt="present an install prompt">presentation of the
```

Comment:
Note, you can put the `data-lt="presentation of the install prompt"` on the `dfn` instead, then you don't need the `data-lt` on the  `a` below.

Reply:
OK I've gone through all references to this in the document and normalized them (in some cases slightly rewording) to avoid data-lts.